skydb.go

```diff
@@ -0,0 +1,102 @@
+package skynet
+
+import (
+	"encoding/hex"
+	"fmt"
+	"gitlab.com/NebulousLabs/errors"
+	"io"
+	"io/ioutil"
+	"math"
+	"os"
+	"strings"
+)
+
+// GetJSON gets the JSON object corresponding to the publicKey and dataKey.
+func (sc *SkynetClient) GetJSON(
+	publicKey string,
+	dataKey string,
+) (io.ReadCloser, error) {
+	entry, err := sc.GetEntry(publicKey, dataKey)
+	if err != nil {
+		return nil, errors.AddContext(err, "could not get entry")
```

Comment:
This shouldn't return an error if the entry wasn't found. In `skynet-js` we return null, I guess this should return nil.